### PR TITLE
TOPAS S3 download

### PR DESCRIPTION
**Pull request by grzanka**
Rebased version of https://github.com/yaptide/yaptide/pull/511

### CODE REVIEW

File: tests/test_download_topas.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        result = False
+    if not 'S3_TOPAS_KEY' in os.environ:
+        result = False
+    if not 'S3_GEANT_BUCKET' in os.environ:
+        result = False
+    if not 'S3_TOPAS_VERSION' in os.environ:
+        result = False
+    return result
+
+def test_if_topas_downloaded(tmpdir):
```
Comment (grzanka) [inline tests/test_download_topas.py:28]: skip this test on Windows

File: tests/test_download_topas.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    """Check if shieldhit is downloaded and can be executed"""
+    if check_if_environment_variables_set():
+        bucket = os.getenv("S3_SHIELDHIT_BUCKET")
+        key = os.getenv("S3_SHIELDHIT_KEY")
```
Comment (grzanka) [inline tests/test_download_topas.py]: shouldn't that be TOPAS ?

File: tests/test_download_topas.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        version = os.getenv("S3_TOPAS_VERSION")
+        geant_bucket = os.getenv("S3_GEANT_BUCKET")
+        assert download_topas_from_s3(topas_bucket_name=bucket, topas_key=key, topas_version=version, geant_bucket_name=geant_bucket, installation_path=tmpdir) is True
+        expected_path = Path(tmpdir / "topas" / key)
```
Comment (grzanka) [inline tests/test_download_topas.py:36]: check unpacking TAR archive and running topas command

**Comment by Bogdan2423** [comment]
https://github.com/yaptide/yaptide/pull/620

**Pull request closed without merging**